scripts/benchmarks/README.md

```diff
@@ -4,10 +4,10 @@ Utilities for benchmarking Ruff.
 
 ## Getting Started
 
-Run `./scripts/benchmarks/run.sh` to clone the benchmarking target (CPython).
+Run `./scripts/benchmarks/setup.sh` to clone the benchmarking target (CPython).
 
-If you're looking to benchmark Ruff against other tools, you'll also need to run `poetry install`
-to create a virtual environment with the required dependencies.
+If you're looking to benchmark Ruff against other tools, you'll also need to run `uv venv --project ./scripts/benchmarks`, 
+activate the venv and finally `uv sync ./scripts/benchmarks` to create a virtual environment with the required dependencies.
```

Comment:
The `uv sync ./scripts/benchmarks` doesn't work. Did you mean to use `--project` flag? Although, even after that it throws an error:
```console
$ uv sync --project ./scripts/benchmark
Using CPython 3.8.18 interpreter at: /Users/dhruv/.local/bin/python3.8
error: The Python request from `scripts/.python-version` resolved to Python 3.8.18, which is incompatible with the project's Python requirement: `>=3.11`. Use `uv python pin` to update the `.python-version` file to a compatible version.
```

Removing the `./scripts/.python-version` does make the command run but that creates the virtual environment in `./scripts` directory while the `uv venv --project ./scripts/benchmarks` command creates the environment in `./scripts/benchmarks` directory.